darts/timeseries.py

```diff
@@ -304,6 +304,30 @@ def copy(self, deep: bool = True) -> 'TimeSeries':
         else:
             return TimeSeries(self._df, self.freq_str())
 
+    def split(self, proportion: float) -> Tuple['TimeSeries', 'TimeSeries']:
+        """
+        Splits he TimeSeries in two, based on a given proportion.
+
+        The proportion must be between 0.0 to 1.0, otherwise an exception will be raised.
+
+        Parameters
+        ----------
+        proportion
+            A float indicating the proportion of the series at which the split is done.
+
+        Returns
+        -------
+        Tuple[TimeSeries, TimeSeries]
+            A tuple of two time series. The first time series contains the first samples up to the `proportion` ,
+            and the second contains the remaining ones (1-`proportion`).
+        """
+        if (proportion < 0) or (proportion > 1):
+            raise_log(ValueError('Proportion must be between 0.0 and 1.0'))
+
+        split_idx = round(len(self)*proportion)
```

Comment:
I think you should rely on `utils.get_timestamp_at_point()` here - in fact, you could even have your method accept `Union[pd.Timestamp, float, int]`, which would be quite nice.

Reply:
I agree! See my response in the next comment.